packages/entities/entities-plugins/src/locales/en.json

```
@@ -21,6 +21,7 @@
     "title": "Configuration",
     "message": "Export configurations for core Kong Gateway objects, like services, routes, and plugins, to either YAML or JSON. This allows for straightforward importing into APIOps tools like decK with YAML, or Insomnia and cURL with JSON.",
```

@tehkai @Miko Bautista Looks the copy in the slideout's description should be updated to include Terraform. 
```suggestion
    "message": "Export configurations for core Kong Gateway objects, like services, routes, and plugins, to either YAML, JSON, or Terraform. This allows for straightforward importing into APIOps tools like decK with YAML, Insomnia and cURL with JSON, or Terraform products with Terraform.",
```